controls/datagrid/filtering/control-template.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Control Template
+page_title: .NET MAUI DataGrid Documentation - Control Template
+description: "Review the Telerik .NET MAUI DataGrid Control Template documentation article to learn more about appliying filtering to the datagriа using FilterControlTemplate property.
```

Comment:
more details in the description
Review the Telerik .NET MAUI DataGrid **Filter** Control Template documentation article to learn more about appliying(typo)> more about how to create custom filtering UI.